.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,35 @@
+name: docker-publish
+on:
+  release:
+    types: [created]
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.13
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Get tools dependencies
+        run: make tools
+
+      - name: Build
+        run: make build
+
+      - name: Push to Docker Hub
```

Comment:
I think `Login docker` can represent the behavior a little better. What do you think?

Reply:
@dc7303 👍